test/functional/apps/dashboard_elements/controls/control_group_chaining.ts

```diff
@@ -121,38 +121,41 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         'grr',
       ]);
       await dashboardControls.ensureAvailableOptionsEqual(controlIds[2], {
-        suggestions: { ...suggestions, grr: suggestions.grr - 1 },
-        invalidSelections: [],
+        ...suggestions,
+        grr: suggestions.grr - 1,
```

Comment:
I cleaned up the logic for `ensureAvailableOptionsEqual` so that it better aligns with its name - i.e. it previously would also check for invalid selections but, in most cases, this was just an empty array anyway. So I separated out the invalid selections check to be a separate function, which I think is a lot cleaner :)